payments-model/res/drawable-night/stripe_ic_unknown_brand_unpadded.xml

```diff
@@ -0,0 +1,16 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
nit - is this a Sail icon? if yes, can we try to keep a pattern when naming these so that we know they're part of our semantics? something like stripe_ic_sail_[name_on_sail].